src/Compilers/VisualBasic/Portable/Symbols/Symbol_Attributes.vb

```diff
@@ -198,6 +198,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             If arguments.Attribute.IsTargetAttribute(Me, AttributeDescription.SkipLocalsInitAttribute) Then
                 arguments.Diagnostics.Add(ERRID.WRN_AttributeNotSupportedInVB, arguments.AttributeSyntaxOpt.Location, AttributeDescription.SkipLocalsInitAttribute.FullName)
             End If
+
+            If arguments.Attribute.IsTargetAttribute(Me, AttributeDescription.ModuleInitializerAttribute) Then
```

Comment:
>arguments.Attribute.IsTargetAttribute(Me, AttributeDescription.ModuleInitializerAttribute) [](start = 15, length = 90)

Should we limit this check to MethodSymbols? If I remember correctly C# ignores the attribute on anything, but a method.

Reply:
Yes, I think that would be more consistent.